src/webvr-polyfill.js

```diff
@@ -107,7 +107,7 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
   navigator.getVRDisplays = this.getVRDisplays.bind(this);
 
   // Polyfill native VRDisplay.getFrameData
-  if (this.nativeWebVRAvailable && window.VRFrameData) {
+  if (this.nativeWebVRAvailable && this.isCardboardCompatible() && window.VRFrameData) {
```

Comment:
I'm not sure why this broke things. I'll take a look...

No, I don't think `nativeWebVRAvailable` should consider `isCardboardCompatible()`.

The reason `VRFrameData` is polyfilled even if it exists is explained here:
https://github.com/googlevr/webvr-polyfill/pull/197